drivers/sensor/bmi160/bmi160.c

```diff
@@ -898,7 +898,7 @@ int bmi160_init(const struct device *dev)
 		return -EIO;
 	}
 
-	k_busy_wait(100);
+	k_busy_wait(150);
```

Comment:
please document constants, ideally add a definition with a docstring referencing why 150 and not e.g. 157.